src/theme/constants.ts

```diff
@@ -7,3 +7,5 @@ export const DEFAULT_BG_COLOR = '#161B32';
 
 export const PRIMARY_MAIN_COLOR = '#252D48';
 export const SECONDARY_MAIN_COLOR = '#A3A7B0';
+
+export const BAR_CHART_WIDTH = 2000;
```

Comment:
Is this ever going to be used outside of BarChart component? If not, a better idea would be to put this inside that module to put it as close to where it's used as possible and reduce polution.